### ISSUE TITLE
Should queryables link header be required?

### ISSUE THREAD

**Issue by aznan2** [opengeospatial/ogcapi-features#905]
RC1 of Part 3, requirements class Queryables, requirement 1-B states: `The link SHALL be included as a Link header in the response.`. However, [Part 1, requirements class Core, Link headers](https://docs.ogc.org/is/17-069r4/17-069r4.html#_link_headers) makes link headers a recommendation only.

It's odd that the queryables link is the only one that requires a header, no? Should this requirement be aligned with the Core spec and be downgraded to a recommendation?

**Comment by cportele** [comment]
Good question, but you have a point. I did not find documentation why we included a requirement for the Link header, so my guess is that we wanted it to be available also when a representation cannot express links in the payload. That said, I agree, to be consistent with the recommendation in Part 1 it should "just" be a recommendation. Actually, since we already have the recommendation in Part 1, we do not need it here, it would be redundant.

Any other views? @pvretano?
